examples/compute_noop.py

```diff
@@ -78,6 +100,10 @@ def compute_shader(input, buffer):
 compute_pass.endPass()
 device.defaultQueue.submit([command_encoder.finish()])
 
-data = buffer2.mapRead()
-# data = await buffer2.mapReadAsync()
-print(np.frombuffer(data, np.int32))
+# Read result
+result = buffer2.mapRead()
+result = IntArrayType.from_buffer(result)  # cast
+print(result[:])
+
+# With Numpy this would be:
+# print(np.frombuffer(buffer2.mapRead(), np.int32))
```

Comment:
Can you add such a comment for the short version too?

Reply:
done